core/wallet-store/src/StoreInternal.test.ts

```diff
@@ -29,9 +29,13 @@ describe('StoreInternal', () => {
             primary: false,
             partyId: 'party1',
             hint: 'hint',
-            fingerprint: 'fp',
-            address: { publicKey: 'pk', privateKey: 'sk' },
-            chainId: 'chain1',
+            signingProviderId: 'internal',
+            publicKey: 'publicKey',
+            namespace: 'namespace',
+            networkId: 'network1',
+            // fingerprint: 'fp',
+            // address: { publicKey: 'pk', privateKey: 'sk' },
+            // chainId: 'chain1',
```

Comment:
wasnt sure which fields we ultimately want or not, so left these commented out for now